Source/Core/Core/PatchEngine.cpp

```diff
@@ -196,9 +196,6 @@ static void ApplyPatches(const std::vector<Patch> &patches)
 				case PATCH_32BIT:
 					Memory::Write_U32(value, addr);
 					break;
-				default:
```

Comment:
What happens when someone generates a patch type in the INI that we don't support?
Should this be a message saying so instead of silently failing?